src/features/access_control/use_cases/get_acl_use_case.py

```diff
@@ -1,10 +1,14 @@
-from authentication.models import AccessControlList, User
-from services.document_service import DocumentService
+from authentication.models import AccessControlList
+from domain_classes.document_look_up import DocumentLookUp
+from storage.data_source_class import DataSource
+from storage.internal.data_source_repository import DataSourceRepository
 
 
-def get_acl_use_case(user: User, document_id: str, data_source_id: str):
-    document_service = DocumentService(user=user)
-    acl: AccessControlList = document_service.get_access_control_list(
-        data_source_id=data_source_id, document_id=document_id
-    )
-    return acl
+def get_acl_use_case(
```

Comment:
duplicate of changes in https://github.com/equinor/data-modelling-storage-service/pull/533/files